config/operator/crd/bases/scylla.scylladb.com_scyllaclusters.yaml

```diff
@@ -15,1701 +15,1792 @@ spec:
-                - location
-                - name
-                type: object
-              type: array
-            cpuset:
-              description: CpuSet determines if the cluster will use cpu-pinning for max performance.
-              type: boolean
-            datacenter:
-              description: Datacenter that will make up this cluster.
-              properties:
-                name:
-                  description: Name of the Scylla Datacenter. Used in the cassandra-rackdc.properties file.
-                  type: string
-                racks:
-                  description: Racks of the specific Datacenter.
-                  items:
-                    description: RackSpec is the desired state for a Scylla Rack.
-                    properties:
-                      agentResources:
-                        description: AgentResources which Agent container will use.
-                        properties:
-                          limits:
-                            additionalProperties:
-                              anyOf:
-                              - type: integer
-                              - type: string
-                              pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
-                              x-kubernetes-int-or-string: true
-                            description: 'Limits describes the maximum amount of compute resources allowed. More info: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/'
-                            type: object
-                          requests:
-                            additionalProperties:
-                              anyOf:
-                              - type: integer
-                              - type: string
-                              pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
-                              x-kubernetes-int-or-string: true
-                            description: 'Requests describes the minimum amount of compute resources required. If Requests is omitted for a container, it defaults to Limits if that is explicitly specified, otherwise to an implementation-defined value. More info: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/'
-                            type: object
-                        type: object
-                      agentVolumeMounts:
-                        description: AgentVolumeMounts to be added to Agent container.
-                        items:
-                          description: VolumeMount describes a mounting of a Volume within a container.
+                      description: RackSpec is the desired state for a Scylla Rack.
+                      properties:
+                        agentResources:
+                          default:
+                            limits:
```

Comment:
I thought agent resources aren't enforces so we don't need to set limits here, or not?

Reply:
It's removed in other PR, i wanted to have change which directly translates existing default to markers, and in other commit remove it.